code/modules/recycling/sortingmachinery.dm

```diff
@@ -185,6 +188,15 @@
 			wrapped_item.AddComponent(/datum/component/pricetag, sticker.payments_acc, sticker.cut_multiplier)
 		update_appearance()
 
+	if(istype(item, /obj/item/boxcutter))
```

Comment:
```suggestion
	else if(istype(item, /obj/item/boxcutter))
```

Otherwise `..()` will hit for all the other items.

Also I kind of hate that this entire proc looks like this but probably not worth it for you to change right now.